third_party/amd/language/hip/utils.py

```diff
@@ -0,0 +1,20 @@
+from triton.language import core
+
+
+@core.extern
+def memrealtime(_semantic=None):
```

Comment:
does it have the same semantic as `tl.extra.cuda.globaltimer` if so can we give it the same name

Reply:
`tl.extra.cuda.globaltimer` is guaranteed to be in nanoseconds: https://docs.nvidia.com/cuda/parallel-thread-execution/index.html#special-registers-globaltimer

For AMD `memrealtime`, it's a timer counter based on a fixed frequency (typically 100MHz ==> 10 ns).